plugin.video.catchuptvandmore/resources/lib/enum/LICENSE

```diff
@@ -0,0 +1,32 @@
+Copyright (c) 2013, Ethan Furman.
```

Comment:
this is the same issue I flagged in the Matrix version.  This module needs to be submitted separately in a Kodi addon wrapper.

Reply:
Done. Thanks fir the feedback